osv/impact.py

```diff
@@ -264,13 +269,14 @@ def _get_commit_to_tag_mappings(repo: pygit2.Repository):
   return mappings
 
 
-def get_commit_and_tag_list(repo,
-                            start_commit,
-                            end_commit,
-                            commits_to_tags=None,
-                            include_start=False,
-                            include_end=True,
-                            limit_commit=None):
+def get_commit_and_tag_list(bug_id: str,
```

Comment:
This is technically a public interface as part of the Python OSV library that we can't change.